src/tools/compiletest/src/header.rs

```diff
@@ -856,7 +864,7 @@ fn iter_header(
             return;
         }
 
-        let Some((header_revision, non_revisioned_directive_line)) = line_directive(comment, ln)
+        let Some((header_revision, non_revisioned_directive_line)) = line_directive("//@", ln)
```

Comment:
It turns out that `line_directive` is also called by debugger tests to parse its own “directives” with a `//` prefix. But I hope to be able to eventually split that off to use its own separate parsing function.